R/les_kb_v2.R

```diff
@@ -300,15 +300,13 @@ utvid_statusvariabel = function(kb_mellom) {
 #' @return kodebok på standardformat
 #' @export
 konverter_oqr_kb = function(kb_mellom) {
-  kb_mellom = oqr_til_std_variabeltyper(kb_mellom)
-  kb_std = velg_standardkolonner(kb_mellom)
-  kb_std = tildel_unike_skjemanavn_fra_skjema_id(kb_std)
-
-  # Ordne rekkefølge for variabler slik variabler fra samme tabell kommer samlet
-  kb_std = kb_std %>%
+  kb_mellom |>
+    oqr_til_std_variabeltyper() |>
+    velg_standardkolonner() |>
+    tildel_unike_skjemanavn_fra_skjema_id() |>
+    # Ordne rekkefølge for variabler slik at
+    # variabler fra samme tabell kommer samlet
```

Comment:
Er det ikkje litt rart/stygt å ha kommentarar midt inni eit røyrsystem?

Reply:
Alternativet er ei (for) lang linje med kommentaren til høgre for røyret? Etter at det må vera oppdelt slik det var opprinneleg?

Reply:
Kommentaren var vel heller ikkje nødvendig, så eg fjerna han.